docs/getting_started.md

```diff
@@ -254,13 +252,20 @@ Make sure your webapp has the credentials to pull the image from the Azure Conta
 
 ![build](./images/multi-stage-webapp.png)
 
+# Customizing the Azure DevOps job container
+
+The pipeline uses a Docker container on the Azure Pipelines agents to accomplish the pipeline steps. The image of the container ***mcr.microsoft.com/mlops/python:latest*** is built with this [Dockerfile](../environment_setup/Dockerfile) and it has all necessary dependencies installed for the purposes of this repository. This image serves as an example of using a custom Docker image that provides a pre-baked environment. This environment is guaranteed to be the same on any building agent, VM or local machine.
+
+In your project you will want to build your own Docker image that only contains the dependencies and tools required for your use case. This image will be more likely smaller and therefore faster, and it will be totally maintained by your team.  See [instructions for customizing the Azure DevOps job container](./custom_container.md).
```

Comment:
I'd like the flow to be to bootstrap next which references the container article.

Reply:
Restored as previous structure, and **added content into the bootstrap README.md**